composer.json

```diff
@@ -6,25 +6,25 @@
     "license": "Apache-2.0",
     "minimum-stability": "stable",
     "require": {
-        "php": ">=7.2,<8.0-dev",
+        "php": ">=7.3,<8.0-dev",
         "ext-dom": "*",
         "ext-openssl": "*",
-        "robrichards/xmlseclibs": "^3.0.4",
-        "simplesamlphp/saml2": "3.2.*",
+        "robrichards/xmlseclibs": "^3.1.1",
+        "simplesamlphp/saml2": "^4.6.4",
         "symfony/dependency-injection": "^4.4",
         "symfony/framework-bundle": "^4.4",
         "symfony/templating": "^4.4",
-        "twig/twig": "^2"
+        "twig/twig": "^2 || ^3",
+        "vimeo/psalm": "^4.30"
```

Comment:
Very cool to add `psalm`, but shouldn't this be a dev dependency?

Reply:
That should have never ended up in this PR.. I used it to make sure I didn't introduce any bugs. Will remove it!